shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/ApacheDubboServiceBeanListener.java

```diff
@@ -62,7 +71,45 @@ public ApacheDubboServiceBeanListener(final PropertiesConfig clientConfig,
                                           final ShenyuClientRegisterRepository shenyuClientRegisterRepository) {
         super(clientConfig, shenyuClientRegisterRepository);
     }
-    
+
+    @Override
+    protected List<ApiDocRegisterDTO> buildApiDocDTO(final Class<?> clazz, final Method method) {
+        final String contextPath = getContextPath();
+        String apiDesc = Stream.of(method.getDeclaredAnnotations()).filter(item -> item instanceof ApiDoc).findAny().map(item -> {
+            ApiDoc apiDoc = (ApiDoc) item;
+            return apiDoc.desc();
+        }).orElse("");
+        String superPath = buildApiSuperPath(clazz, AnnotatedElementUtils.findMergedAnnotation(clazz, getAnnotationType()));
+        if (superPath.indexOf("*") > 0) {
+            superPath = superPath.substring(0, superPath.lastIndexOf("/"));
+        }
+        ShenyuDubboClient shenyuDubboClient = AnnotatedElementUtils.findMergedAnnotation(method, ShenyuDubboClient.class);
+        if (Objects.isNull(shenyuDubboClient)) {
+            return Lists.newArrayList();
+        }
+        List<ApiDocRegisterDTO> list = Lists.newArrayList();
+        String value = shenyuDubboClient.value();
+        String apiPath = contextPath + superPath + value;
+        ApiDocRegisterDTO build = ApiDocRegisterDTO.builder()
+                .consume(ShenyuClientConstants.MEDIA_TYPE_ALL_VALUE)
+                .produce(ShenyuClientConstants.MEDIA_TYPE_ALL_VALUE)
+                .httpMethod(ApiHttpMethodEnum.NOT_HTTP.getValue())
+                .contextPath(contextPath)
+                .ext("{}")
+                .document("{}")
+                .version("v0.01")
```

Comment:
In `dubbo`, maybe we can build version from `ServiceBean`.

Reply:
i try this code,but it didn't work
```java

if (apiModuleClass.isAnnotationPresent(Service.class)) {
    Service dubboService = apiModuleClass.getAnnotation(Service.class);
    apiVersion = dubboService.version();
} else {
    DubboService dubboService = apiModuleClass.getAnnotation(DubboService.class);
    apiVersion = dubboService.version();
}

```

Reply:
I mean `org.apache.dubbo.config.spring.ServiceBean#getVersion`

Reply:
in `alibaba dubbo` or `apache dubbo`. it's seem look like empty.

![image](https://user-images.githubusercontent.com/12622645/209522023-4e98f563-2b31-410e-bba7-1b766186f486.png)

![image](https://user-images.githubusercontent.com/12622645/209522035-04b67540-6ec7-4e0c-a07b-458520d0e7aa.png)

Reply:
try config it in `<dubbo:service version="1.0.0">` in xml, or `@DubboService(version="1.0.0")` in code.

Reply:
i will try